lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -133,15 +133,16 @@ def self.commmit_changes_and_push_current_branch(commit_message)
         Actions::PushToGitRemoteAction.run(remote: 'origin')
       end
 
-      def self.create_pr_to_main(title, body, repo_name, github_pr_token)
+      def self.create_pr_to_main(title, body, repo_name, github_pr_token, labels)
```

Comment:
Maybe set `nil` as default value? That said, since we are only using it in these 2 actions, I think it's ok for now.

Reply:
I set this to default to `[]` 👍🏻